.goreleaser-cdk.yaml

```diff
@@ -10,7 +10,8 @@ before:
   hooks:
     - go mod download
     - go install github.com/gobuffalo/packr/v2/packr2@v2.8.3
-    - ( cd db && packr2 )
+    - cd db
+    - packr2
```

Comment:
I was under the impression that this command would not be run in the db directory, but maybe it just works?

Reply:
just works, I checked 🤷

Reply:
awesome, I did too, and I think we can drop the `cd db` part, and then can we just .gitignore the packr2 generated code?

Reply:
nono, without the `cd db` it doesn't work, I'll merge this as is, gen image and open a PR in node replacing packr2